src/base/datetime/calendar/header/index.ts

```diff
@@ -0,0 +1,217 @@
+import { html, property, PropertyValues } from "lit-element";
+import {
+  KucBase,
+  dispatchCustomEvent,
+  CustomEventDetail
+} from "../../../kuc-base";
+import {
+  getLeftArrowIconSvgTemplate,
+  getLocale,
+  getRightArrowIconSvgTemplate
+} from "../header/ultils";
+import "../../calendar/header/dropdown/month";
+import "../../calendar/header/dropdown/year";
+import "../../listbox";
+import { en } from "../../resource/locale";
+
+export class BaseDateTimeCalendarHeader extends KucBase {
+  @property({ type: String }) language = "en";
+  @property({ type: Number }) month = 1;
+  @property({ type: Number }) year = 2021;
+
+  private _locale = en;
+
+  update(changedProperties: PropertyValues) {
+    if (changedProperties.has("language")) {
+      this._locale = getLocale(this.language);
+    }
+    super.update(changedProperties);
+  }
+
+  render() {
+    return html`
+      ${this._getStyleTagTemplate()}
+      <div class="kuc-base-datetime-calendar-header__group">
+        <button
+          aria-label="previous month"
+          type="button"
+          class="kuc-base-datetime-calendar-header__group__button kuc-base-datetime-calendar-header__group__button-previous-month"
+          @click=${this._handleClickCalendarPrevMonthBtn}
```

Comment:
Add double quotation.
```suggestion
          @click="${this._handleClickCalendarPrevMonthBtn}"
```